front/src/applications/stdcm/consts.ts

```diff
@@ -7,6 +7,7 @@ export const STDCM_REQUEST_STATUS = Object.freeze({
   rejected: 'REJECTED',
   canceled: 'CANCELED',
   noresults: 'NORESULTS',
+  pendingadditional: 'PENDINGADDITIONAL',
```

Comment:
Maybe this would be easier to read
```suggestion
  pending_additional: 'PENDING_ADDITIONAL',
```